build.gradle.kts

```diff
@@ -233,6 +233,8 @@ dependencies {
             testImplementation(project(path = it.name, configuration = "testArtifacts"))
         }
 
+    implementation("dev.zacsweers.moshix:moshi-metadata-reflect:0.19.0")
```

Comment:
Can't this live in `http4k-format/mochi/build.gradle.kts` rather than the root gradle config?

Reply:
this is a good point - although it is consistent with our other examples. All of the dependencies for the docs live in the root module at the moment. :)

Reply:
Ah! That's the dependency for example! I thought it was the actual one for the mochi module. Please ignore me...